From 4144cce74bde2d287168b788369fdd1afa78235f Mon Sep 17 00:00:00 2001
From: Shunqian Zheng <zhengsq@rock-chips.com>
Date: Mon, 24 Dec 2018 16:28:12 +0800
Subject: [PATCH] arm64: dts: rockchip: remove regulator-always-on for rk3326
 linux evb

DVP powers are controlled by sensor itself, removing always-on
propertity.

Meanwhile, the vcc_sd is boot-on according to schematic.

Change-Id: I74809672b9ec7ae250243e47e7e700236d5b0839
Signed-off-by: Shunqian Zheng <zhengsq@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts
index 81b6ef8e8bca..a6b28fa86f3d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts
@@ -576,6 +576,7 @@
 			vcc_sd: LDO_REG6 {
 				regulator-min-microvolt = <3300000>;
 				regulator-max-microvolt = <3300000>;
+				regulator-boot-on;
 
 				regulator-name = "vcc_sd";
 				regulator-state-mem {
@@ -586,7 +587,6 @@
 			};
 
 			vcc2v8_dvp: LDO_REG7 {
-				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <2800000>;
 				regulator-max-microvolt = <2800000>;
@@ -599,7 +599,6 @@
 			};
 
 			vcc1v8_dvp: LDO_REG8 {
-				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
@@ -612,7 +611,6 @@
 			};
 
 			vdd1v5_dvp: LDO_REG9 {
-				regulator-always-on;
 				regulator-boot-on;
 				regulator-min-microvolt = <1500000>;
 				regulator-max-microvolt = <1500000>;
-- 
2.35.3

